openedx/core/djangoapps/catalog/docs/decisions/002-course-models.rst

```diff
@@ -0,0 +1,76 @@
+Course Models
+#############
+
+Status
+******
+
+Accepted
+
+Decision
+********
+
+edx-platform will have a first-class model to represent the idea of a catalog
+course. If we think of multiple runs of the same course to all derive from one
+conceptiual course.  The catalog course would be that conceptual course.
+
+Context
+*******
+
+Historically this concept has been inferred from the names of courses with the
+runs removed (eg. "AximX/Eng101" could be a catalog course with multiple runs
+(2024H1, 2024H2, etc)). Having this explicitly defined as a new model has
+multiple benefits.
+
+* Relationships between course runs can be defined explicitly instead of implicitly.
+
+  * Currently this information is either implied or we require the entire
+    course-discovery service in order to retrieve this information.
+
+* A list of all courses can be easily queried both internally and via the rest API.
+
+* Courses can be explicitly mapped to organizations rather than implicitly.
+
+* This change opens up the door for program or course bundle data to be persisted
+so that a reset of mem cache doesn't break programs related features.
```

Comment:
make this present tense, like the rest of the bullets.

```suggestion
* The door is opened to also persist program or course bundle data in the future.
  This will stop programs-related features from breaking whenever memcached is reset.
```